docs/user/alerting/defining-alerts.asciidoc

```diff
@@ -74,6 +74,21 @@ image::images/alert-flyout-add-action.png[You can add multiple actions on an ale
 Actions are not required on alerts. In some cases you may want to run an alert without actions first to understand its behavior, and configure actions later.
 ==============================================
 
+[float]
+=== Global actions configuration
+Some actions configuration options apply to all actions.
+If you are using an *on-premises* Elastic Stack deployment, you can set these in the kibana.yml file.
```

Comment:
Although "on-premises" and "on-prem" are both allowed, I think "on-prem" sounds better here.

Reply:
done